src/Sylius/Bundle/ShopBundle/templates/shared/address/country.html.twig

```diff
@@ -0,0 +1 @@
+<span>{{ hookable_metadata.context.address.countryCode|sylius_country_name|upper }}</span>
```

Comment:
Shouldn't the country code be declared at the top? like
`{% set countryCode = hookable_metadata.context.address.countryCode %}`. 
This applies to this and other files